package.json

```diff
@@ -37,6 +37,6 @@
     "typescript": "4.7.3"
   },
   "lint-staged": {
-    "**/*": "prettier --write --ignore-unknown"
+    "**/*.{js,ts,jsx,tsx}": "prettier --write --ignore-unknown"
```

Comment:
Prevent prettier from formatting Markdown files.

Reply:
Yeah, basically the markdown renders weirdly in some scenarios for formatted Markdown. More infor [here](https://github.com/mirainc/developer-docs/pull/12#discussion_r903321362).